nullaway/src/main/java/com/uber/nullaway/fixserialization/FixSerializationConfig.java

```diff
@@ -89,6 +89,8 @@ public FixSerializationConfig(
     serializer = new Serializer(this, initializeAdapter(SerializationAdapter.LATEST_VERSION));
   }
 
+
+
```

Comment:
Can you run our configured formatter (`./gradlew spotlessApply`) on this PR?

We try to be consistent in formatting and these two blank lines don't seem right.